src/Contract/Config/DeptracConfig.php

```diff
@@ -21,13 +21,22 @@ final class DeptracConfig implements ConfigBuilderInterface
     private array $formatters = [];
     /** @var array<Ruleset> */
     private array $rulesets = [];
+    /** @var ?string */
+    private ?string $internalTag = null;
     /** @var array<string, EmitterType> */
     private array $analyser = [];
     /** @var array<string, array<string>> */
     private array $skipViolations = [];
     /** @var array<string> */
     private array $excludeFiles = [];
 
+    public function internalTag(?string $tag): self
```

Comment:
Why is it a standalone function and not a parameter to `public function analysers`? It is incongruent with how it is specified in the `yaml` config.

Reply:
The analysers() function is already a bit odd... it sets the "types" key in the "analyser" section, not the entire section. But sure, we could have one parameter for each key in that section.

Keep in mind we will add another one soon, for skipDeprecated. I generally dislike long lists of parameters. But since PHP 8 has named params, perhaps I should learn to love them.

Alternatively, there could be a separate AnalyzerConfig object, and internalTag() and skipDeprecated() could me methods on that object...

Reply:
`AnalyzerConfig` seems like the logical consistent way to go to me.

Reply:
done